src/Upscale.hpp

```diff
@@ -107,39 +107,25 @@ class Upscale : public mfem::Operator
     virtual void OrthogonalizeCoarse(mfem::Vector& vect) const;
     virtual void OrthogonalizeCoarse(mfem::BlockVector& vect) const;
 
-    /// Create a coarse vertex space vector
-    virtual mfem::Vector GetCoarseVector() const;
+    virtual void OrthogonalizeLevel(int level, mfem::Vector& vect) const;
```

Comment:
Like `GetMatrix`, `GetVector`, etc. in this class, can we collapse each of `Orthogonalize`, `Restrict`, `Interpolate`, `BlockOffsets` and possibly `Solve` to only one that takes `level` as an input argument.